### DISCUSSION TITLE
Can Someone help to Import Google earth in to QGIS3.22

### DISCUSSION THREAD

**Discussion by Siva1992dm** [gee-community/qgis-earthengine-plugin#110] [Q&A]
I Have installed Google earth engine plugin in to QGIS,

But am unable to write the script and run for the Google earth
![GEE_20220606](https://user-images.githubusercontent.com/105856875/172118762-60b11991-4fa6-4bb2-aa73-116cbe6a4730.JPG)

**Comment by XavierCLL** [comment]
Hi, 

If you cannot write in the python editor that is a Qgis issue (not the plugin) try to uninstall it and install the latest one (Qgis 3.24)
